src/document/change/change.ts

```diff
@@ -23,13 +23,24 @@ import {
 import { CRDTRoot } from '@yorkie-js-sdk/src/document/crdt/root';
 import { ChangeID } from '@yorkie-js-sdk/src/document/change/change_id';
 import { Indexable } from '@yorkie-js-sdk/src/document/document';
+import { converter } from '@yorkie-js-sdk/src/api/converter';
 import { HistoryOperation } from '@yorkie-js-sdk/src/document/history';
 import {
   PresenceChange,
   PresenceChangeType,
 } from '@yorkie-js-sdk/src/document/presence/presence';
 import { deepcopy } from '@yorkie-js-sdk/src/util/object';
 
+export type ChangeStruct = {
```

Comment:
Please also add comments. and is there a reason why it is difficult to specify `ChangeStruct<T>`?